agent/test-agent-cli/examples/example_test_agent_cli/example-test.sh

```diff
@@ -0,0 +1,71 @@
+# Get the configuration details for test and save it in a JSON file
+test-agent-cli get config > ExampleConfig.json
+
+echo "starting the Example test"
+
+#Use JQ to read the config JSON
+#https://www.linode.com/docs/guides/using-jq-to-process-json-on-the-command-line/
+person=$(jq '.person' ExampleConfig.json)
+helloCount=$(jq '.helloCount' ExampleConfig.json)
+helloDurationMilliseconds=$(jq '.helloDurationMilliseconds' ExampleConfig.json)
+passed=0
+
+echo "$person"
+echo "$helloDurationMilliseconds"
+echo "$helloCount"
+
+# Set the Task_state as running(This can be checked in Kubectl describe test or testsys status)
+test-agent-cli send test start
+
```

Comment:
Consider having get config signal the start of a test. Maybe `test-agent-cli init` which returns the config.